include/nuttx/net/netdev.h

```diff
@@ -242,7 +242,7 @@ struct net_driver_s
 
   /* Drivers interface flags.  See IFF_* definitions in include/net/if.h */
 
-  uint8_t d_flags;
+  uint32_t d_flags;
```

Comment:
Just a question: why type is expanded to 32bits and not to 16bits?

Reply:
Refer to the definition of Linux, reserve space for further expansion